debian/install

```diff
@@ -1,3 +1,4 @@
 ubuntu-advantage usr/bin/
 keyrings/*.gpg usr/share/keyrings/
+modules/* usr/lib/ubuntu-advantage/modules
```

Comment:
Do you know what's the difference between /usr/share/ubuntu-advantage and /usr/lib/ubuntu-advantage? At first I thought /usr/lib was reserved for arch-dependent files, and /usr/share/ for the rest, so I would advocate for the ua modules to be under /usr/share. But python is in /usr/lib, but (again) it has .pyc files which I think are arch-dependent.

Reply:
I don't really know the policy there, I always thought `lib` is for "library" files, so code that can be considered modules (hence my choice here), whereas `share` would be for other type of data files.

If we are to believe https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard, I would think modules could fit the definition of "Libraries for the binaries in /usr/bin and /usr/sbin.", so they could fit in `/usr/lib`.

Anyway, I'm fine on changing it to `/usr/share` if it's better suited.

Reply:
https://www.debian.org/doc/debian-policy/#file-system-hierarchy, first item of that list, makes me think `/usr/share` is more appropriate:

> The FHS requirement that architecture-independent application-specific static files be located in /usr/share is relaxed to a suggestion. In particular, a subdirectory of /usr/lib may be used by a package (or a collection of packages) to hold a mixture of architecture-independent and architecture-dependent files. However, when a directory is entirely composed of architecture-independent files, it should be located in /usr/share.

Reply:
Ok, that seems more authoritative, done :)